src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let year = '';
+  let month = '';
+  let day = '';
+
+  const oldFormat = date.split(fromFormat[3]);
```

Comment:
It'd be better to create variable for `fromFormat[3]`, e.g. `oldSeparator`
```suggestion
  const oldFormat = date.split(fromFormat[3]);
```

Reply:
Fixed